azerothcore/mod-npc-buffer
# Fix SQL For creature_template and cleanup

## Changes Proposed:
- Update SQL

## Issues Addressed:
Fixes failure and issues related to changes to Creature_Template


## Tests Performed:
- Tested on Current Standard Build and Also Playerbots


## How to Test the Changes:
1. Install Module
2. Should be no errors upon compile and install
3. SQL should import without errors
4. Summon NPC (Creature ID 601016)
5. Right click to speak to NPC - He should Emote and Buff you.
